fetch/connection-pool/resources/network-partition-key.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import mimetypes
+import os
+import sys
+import time
+
+# Test server that tracks the last partition_id was used with each connection for each uuid, and
+# lets consumers query if multiple different partition_ids have been been used for any socket.
+def main(request, response):
+    response.headers.set(b"Cache-Control", b"no-store")
+    dispatch = request.GET.first(b"dispatch", None)
+    uuid = request.GET.first(b"uuid", None)
+    partition_id = request.GET.first(b"partition_id", None)
+
+    if not uuid or not dispatch or not partition_id:
+        return simple_response(request, response, 404, b"Not found", b"Invalid query parameters")
+
+    # Unless nocheck_partition is true, check partition_id against server_state, and update server_state.
+    stash = request.server.stash
+    test_failed = False
+    if (request.GET.first(b"nocheck_partition", None) != "True"):
```

Comment:
Nit; no outer parentheses needed in Python here, I believe.

Reply:
Done.  Thanks!